app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,24 @@
+class GroupsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
we want to keep the whole app behind sign_in wall, so don't skip authentication

Reply:
I'm having trouble signing up. Is there some validation setting on devise or pundit rn preventing new sign ups? 
![image](https://user-images.githubusercontent.com/3223329/112416244-a76d3780-8d68-11eb-82df-f68bac74ea03.png)

Reply:
best to check logs in rails server, what is telling you